src/SymCrypt-OpenSSL-Debian/Makefile

```diff
@@ -9,17 +9,23 @@ BUILD_NAME = symcrypt-openssl
 SYMCRYPT_OPENSSL_VERSION ?= 0.1
 SYMCRYPT_PACKAGE = $(BUILD_NAME)_$(SYMCRYPT_OPENSSL_VERSION)_$(ARCH).deb
 BUILD_ROOT_DIR = tmp/$(BUILD_NAME)
+SYMCRYPT_TEST ?= y
 
 CMAKE_ARCH = AMD64
+SYMCRYPT_USE_ASM = ON
 LIB_INSTALL_NAME = x86_64-linux-gnu
 ifeq ($(ARCH), arm64)
 CMAKE_ARCH = ARM64
 LIB_INSTALL_NAME = aarch64-linux-gnu
 else ifeq ($(ARCH), armhf)
-CMAKE_ARCH = ARMHF
-LIB_INSTALL_NAME = arm-linux-gnueabihf 
+CMAKE_ARCH = ARM
+LIB_INSTALL_NAME = arm-linux-gnueabihf
+SYMCRYPT_USE_ASM = OFF
```

Comment:
Why this? The armhf SymCrypt should work with asm enabled

Reply:
Not convinced that these SYMCRYPT_USE_ASM lines do anything; the variables are not referenced anywhere else (i.e. they not passed to cmake or build.py).

The default will have asm on, which is correct, and looking at the logs from the pipeline it looks like the default is being used; probably just worth removing these line to avoid confusion.

Reply:
The config is no use, removed, thanks.